example/src/test/scala/monocle/function/AtExample.scala

```diff
@@ -34,4 +38,21 @@ class AtExample extends MonocleSuite {
     (Set(1, 2, 3) applyLens at(2) set None)     shouldEqual Set(1, 3)
   }
 
+  test("at creates a Lens from Int to one of its bit") {
+    (3 applyLens at(0: IntBits) get) shouldEqual true   // true  means bit is 1
+    (4 applyLens at(0: IntBits) get) shouldEqual false  // false means bit is 0
+
+
+    (32 applyLens at(0: IntBits) set true)   shouldEqual 33
+    (3  applyLens at(1: IntBits) modify(!_)) shouldEqual 1 // toggle 2nd bit
+
+    illTyped("""0 applyLens at(79) get""")
```

Comment:
Ha! In the passing examples above the ascription `: IntBits` is necessary to guide the implicit search. So this example probably fails because it can't find a suitable `At` instance and not because `79` is not a valid `IntBits`. 

Matching against the expected compile error would have caught that. :smiley: